api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -320,6 +320,7 @@ class Config:
 class GetIndividualOfferResponseModel(BaseModel, AccessibilityComplianceMixin):
     activeMediation: GetOfferMediationResponseModel | None
     bookingContact: str | None
+    bookingsCount: int = 0
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏Pas besoin du `= 0` ici
On attend un `int`, la fonction `get_bookings_count_by_offer` va retourner un `int`, pas besoin de valeur à 0 par défaut